scrapy/commands/crawl.py

```diff
@@ -1,9 +1,8 @@
-from scrapy.commands import ScrapyCommand
-from scrapy.utils.conf import arglist_to_dict, feed_process_params_from_cli
 from scrapy.exceptions import UsageError
+from scrapy.commands import BaseRunSpiderCommands
```

Comment:
Let’s keep it alphabetical.

```suggestion
from scrapy.commands import BaseRunSpiderCommands
from scrapy.exceptions import UsageError
```